rules/framework.bzl

```diff
@@ -172,6 +172,10 @@ def _apple_framework_packaging_impl(ctx):
 
             # collect modulemaps
             for modulemap in dep[apple_common.Objc].direct_module_maps:
+                if modulemap.owner == dep.label:
```

Comment:
to make sure I understand this correctly, for direct module maps of a given obj-c dependency, if the module map is generated by the dep itself, including it will cause redefinition error, so skip them at this point is the way to fix this error

Reply:
yes